### PR TITLE
Add Script for Generating UPER-Encoded Hex Strings from ODE TIM JSON Files

### LINKED ISSUE #2: Script for Generating ASN1 Hex Strings from JSON TIM Requests
A script should be provided to enable users to generate ASN1 hex strings from TIMs described in JSON request files processed by the ODE.

### PR DESCRIPTION
**Pull request by dmccoystephenson**
## Problem
Users need a simple way to generate TIMs without manually searching through ODE output logs.

## Solution
A script has been added that allows users to generate UPER-encoded hex strings directly from ODE TIM JSON request files.

## Testing
The solution was tested locally using WSL.

## Relevant Issues
Closes #2

## Dependent PR
This PR relies on the changes in #3 and should be merged after that one.

### CODE REVIEW

File: data/tim.uper [hunk 1]
```diff
@@ -0,0 +1 @@
+001F80A97014B901EC9C236B00000000000F775D9B0301EA73E452D1539716C99E9D555100003F0A59B080010007F8AA9979F4D3BB3A0A9266C000000854E3B2C47291F21E85EEF057980028422C1FFE0001FFFC00017FFF80000FFFFF000009FFFFF8000005FFFFFF00000007FF80007FFF00005FFFE00003FFFFC000027FFFFE0000017FFFFFC000000C523E43D138ECB11C0000C00FB0473DFB72A0E997C74007D408E5C376CD4F775D9B00
```
Comment (payneBrandon) [inline data/tim.uper]: Let's remove and add a .gitignore entry for the uper so we can test it

File: generate_tim.sh [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+if [ $# -ne 1 ]; then
+    echo "Usage: ./generate_tim.sh <filename>"
+    exit 1
+fi
+
+filename=$1
+
+# retrieve IP address of the host machine
+export DOCKER_HOST_IP=$(ifconfig | grep -A 1 'inet ' | grep -v 'inet6\|127.0.0.1' | awk '{print $2}' | grep -E '^172\.1[6-9]\.|^172\.2[0-9]\.|^172\.3[0-1]\.|^192\.168\.' | head -n 1)
```
Comment (payneBrandon) [inline generate_tim.sh]: Should we try to pull from the .env first in case someone is running this script against a different ip?

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Just the one question with the IP. Otherwise very clean solution, looks great!

File: data/tim.uper (hunk 1 above)
Comment (dmccoystephenson) [reply data/tim.uper]: Removed this file & added a .gitignore entry for uper files

File: generate_tim.sh (hunk 2 above)
Comment (dmccoystephenson) [reply generate_tim.sh]: Yup sounds like a good idea. Modified this script to use the .env if it exists and to create one if it doesn't, just like the other script.

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**